include/device_time_origin.h

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020 The urg_stamped Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef DEVICE_TIME_ORIGIN_H
+#define DEVICE_TIME_ORIGIN_H
+
+#include <ros/ros.h>
+
+#include <cmath>
+
+namespace device_time_origin
+{
+class DriftedTime
+{
+public:
+  ros::Time origin_;
+  double gain_;
+
+  inline DriftedTime()
+    : gain_(1.0)
+  {
+  }
+  inline DriftedTime(const ros::Time origin, const float gain)
+    : origin_(origin)
+    , gain_(gain)
+  {
+  }
+};
+
+namespace estimator
+{
+inline static ros::Time estimateOriginByAverage(
```

Comment:
static (internal linkage) and inline are redundant.
```suggestion
inline ros::Time estimateOriginByAverage(
```

Reply:
done https://github.com/seqsense/urg_stamped/pull/66/commits/7f9f216d067c984fa1a2849f565a6e7d9d90d106